onnxruntime/core/providers/cuda/cuda_call.cc

```diff
@@ -20,6 +20,11 @@ const char* CudaErrString(ERRTYPE) {
   ORT_NOT_IMPLEMENTED();
 }
 
+template <typename ERRTYPE>
+int CudaErrCode(ERRTYPE err_code) {
+    return (int)err_code;
```

Comment:
use static_cast / reinterprest. c-style casts are always evil since they don't tell the reader about the intention are are more unsafe.

Reply:
Maybe rename the function to CudaErrCodeAsInt as well (or use static cast inplace...)